testnets/synternettestnet/chain.json

```diff
@@ -0,0 +1,167 @@
+  },
+  "staking": {
+    "staking_tokens": [
+      {
+        "denom": "uamber"
+      }
+    ]
+  },
+  "codebase": {
+    "git_repo": "https://github.com/Synternet/synternet-chain-releases",
+    "recommended_version": "v0.8.0",
+    "compatible_versions": [
+      "v0.8.0"
+    ],
+    "binaries": {
+      "linux/amd64": "https://github.com/Synternet/synternet-chain-releases/releases/download/v0.8.0/amberd-linux-amd64-v0.8.0-testnet"
+    },
+    "genesis": {
+      "genesis_url": "https://github.com/Synternet/synternet-chain-releases/blob/main/networks/testnet/genesis.json"
+    },
+    "versions": [
+      {
+        "name": "v0.1",
+        "recommended_version": "v0.1.0",
+        "compatible_versions": [
+          "v0.1.0"
+        ],
+        "binaries": {
+          "linux/amd64": "https://github.com/Synternet/synternet-chain-releases/releases/download/v0.0.1/amberd-linux-amd64-v0.1.0-testnet"
+        }
+      },
+      {
+        "name": "v0.2",
+        "recommended_version": "v0.2.4",
+        "compatible_versions": [
+          "v0.2.4"
+        ],
+        "binaries": {
+          "linux/amd64": "https://github.com/Synternet/synternet-chain-releases/releases/download/v0.2.4/amberd-linux-amd64-v0.2.4-testnet"
+        }
+      },
+      {
+        "name": "v0.5",
+        "recommended_version": "v0.5.4",
+        "compatible_versions": [
+          "v0.5.4"
+        ],
+        "binaries": {
+          "linux/amd64": "https://github.com/Synternet/synternet-chain-releases/releases/download/v0.5.4/amberd-linux-amd64-v0.5.4-testnet"
+        }
+      },
+      {
+        "name": "v0.6",
+        "recommended_version": "v0.6.0",
+        "compatible_versions": [
+          "v0.6.0"
+        ],
+        "binaries": {
+          "linux/amd64": "https://github.com/Synternet/synternet-chain-releases/releases/download/v0.6.0/amberd-linux-amd64-v0.6.0-testnet"
+        }
+      },
+      {
+        "name": "v0.7",
+        "recommended_version": "v0.7.0",
+        "compatible_versions": [
+          "v0.7.0"
+        ],
+        "binaries": {
+          "linux/amd64": "https://github.com/Synternet/synternet-chain-releases/releases/download/v0.7.0/amberd-linux-amd64-v0.7.0-testnet"
+        }
+      },
+      {
+        "name": "v0.8",
+        "recommended_version": "v0.8.0",
+        "compatible_versions": [
+          "v0.8.0"
+        ],
+        "binaries": {
+          "linux/amd64": "https://github.com/Synternet/synternet-chain-releases/releases/download/v0.8.0/amberd-linux-amd64-v0.8.0-testnet"
+        }
+      }
+    ]
+  },
+  "peers": {
+    "seeds": [
+      {
+        "id": "639d50339d7045436c756a042906b9a69970913f",
+        "address": "seed-01.theta-testnet.polypore.xyz:26656",
+        "provider": "Hypha"
+      },
+      {
+        "id": "3e506472683ceb7ed75c1578d092c79785c27857",
+        "address": "seed-02.theta-testnet.polypore.xyz:26656",
+        "provider": "Hypha"
+      }
+    ],
+    "persistent_peers": []
```

Comment:
Just add our testnet validators as `persistent_peers` since these are the only genesis validators. You can add one or more external validators though.
We don't have seed peers.

Reply:
Where should I take them from? Genesis.json or some other place?

Reply:
`networks/testnet/README.md` in Gitlab should contain our nodes and external validators, though it only contains our validators.
So the next best place is asking our external validators for the peer address.

Reply:
ok, updated. Injective hasn't added external ones so I also added just ours.